### ISSUE TITLE
inquiry: ECMAScript escape error and missing assets files

### ISSUE THREAD

**Issue by nemu1113** [kwchang0831/svelte-QWER#24]
Hi! This case is a question. Possibly it could be a bug.
Please lend me your help.

First, if you run this template in its default state under a local Windows environment, you will get the following error.

- Error log contents (some parts are omitted)
```pwsh
PS C:\Git\qwer> npm run dev

> @kwchang0831/svelte-qwer@0.3.3 dev 
> npm-run-all -s build:QWER dev:watch

> @kwchang0831/svelte-qwer@0.3.3 build:QWER
> node --no-warnings QWER build

> @kwchang0831/svelte-qwer@0.3.3 dev:watch
> npm-run-all -p -r dev:QWER dev:i18n dev:sveltekit

> @kwchang0831/svelte-qwer@0.3.3 dev:QWER
> node --no-warnings QWER watch

> @kwchang0831/svelte-qwer@0.3.3 dev:i18n
> npx typesafe-i18n

> @kwchang0831/svelte-qwer@0.3.3 dev:sveltekit
> cross-env MODE=development vite dev --host

  VITE v3.1.1  ready in 1753 ms

  ➜  Local:   http://localhost:5173/
  ➜  Network: http://192.168.11.103:5173/

Transform failed with 1 error:
C:/Git/qwer/src/generated/assets.ts:89:8: ERROR: Legacy octal escape sequences cannot be used in an ECMAScript module
Transform failed with 1 error:
C:/Git/qwer/src/generated/assets.ts:89:8: ERROR: Legacy octal escape sequences cannot be used in an ECMAScript module

Legacy octal escape sequences cannot be used in an ECMAScript module
87 |  export const assets = (() => {
88 |    const _data = new Map<string, Asset.Image>([
89 |      ['C:\404.webp', { 800: ASSETS_0, 1280: ASSETS_1, original: ASSETS_2, banner: ASSETS_3, extraFormats: ASSETS_4 }],
   |          ^
90 |      ['C:avatar.png', { 800: ASSETS_5, 1280: ASSETS_6, original: ASSETS_7, banner: ASSETS_8, extraFormats: ASSETS_9 }],
91 |      [

Error: Transform failed with 1 error:
C:/Git/qwer/src/generated/assets.ts:89:8: ERROR: Legacy octal escape sequences cannot be used in an ECMAScript module
    at failureErrorWithLog (C:\Git\qwer\node_modules\.pnpm\esbuild@0.15.7\node_modules\esbuild\lib\main.js:1624:15)
    at C:\Git\qwer\node_modules\.pnpm\esbuild@0.15.7\node_modules\esbuild\lib\main.js:1413:29
    at C:\Git\qwer\node_modules\.pnpm\esbuild@0.15.7\node_modules\esbuild\lib\main.js:678:9
    at handleIncomingPacket (C:\Git\qwer\node_modules\.pnpm\esbuild@0.15.7\node_modules\esbuild\lib\main.js:775:9)
    at Socket.readFromStdout (C:\Git\qwer\node_modules\.pnpm\esbuild@0.15.7\node_modules\esbuild\lib\main.js:644:7)
    at Socket.emit (node:events:513:28)
    at addChunk (node:internal/streams/readable:315:12)
    at readableAddChunk (node:internal/streams/readable:289:9)
    at Socket.Readable.push (node:internal/streams/readable:228:10)
    at Pipe.onStreamRead (node:internal/stream_base_commons:190:23)
```
I was able to resolve the above error with a manual patch, but the following error occurred subsequently.

It is still possible that I do not fully understand sveltekit and typescript, in which case I apologize, but when I run this template here in my Windows local environment, I get the following error and cannot build it.

- Error log contents (some parts are omitted)
```pwsh
PS C:\Git\qwer> npm run dev

> @kwchang0831/svelte-qwer@0.3.3 dev 
> npm-run-all -s build:QWER dev:watch

> @kwchang0831/svelte-qwer@0.3.3 build:QWER
> node --no-warnings QWER build

> @kwchang0831/svelte-qwer@0.3.3 dev:watch
> npm-run-all -p -r dev:QWER dev:i18n dev:sveltekit

> @kwchang0831/svelte-qwer@0.3.3 dev:i18n
> npx typesafe-i18n

> @kwchang0831/svelte-qwer@0.3.3 dev:sveltekit
> cross-env MODE=development vite dev --host

> @kwchang0831/svelte-qwer@0.3.3 dev:QWER
> node --no-warnings QWER watch

  VITE v3.1.1  ready in 2171 ms

Input file is missing: /src/generated/assetsC:avatar.png
Input file is missing: /src/generated/assetsC:avatar.png (x2)
Input file is missing: /src/generated/assetsC:avatar.png (x3)
Input file is missing: /src/generated/assetsC:avatar.png (x4)
Input file is missing: /src/generated/assetsC:default_og_card.webp
Input file is missing: /src/generated/assetsC:default_og_card.webp (x2)
Input file is missing: /src/generated/assetsC:default_og_card.webp (x3)
Input file is missing: /src/generated/assetsC:default_og_card.webp (x4)
Input file is missing: /src/generated/assetsC:mobile-preview.png
Input file is missing: /src/generated/assetsC:mobile-preview.png (x2)
Input file is missing: /src/generated/assetsC:mobile-preview.png (x3)
Input file is missing: /src/generated/assetsC:mobile-preview.png (x4)
Input file is missing: /src/generated/assetsC:mobile-preview.webp
Input file is missing: /src/generated/assetsC:mobile-preview.webp (x2)
Input file is missing: /src/generated/assetsC:mobile-preview.webp (x3)
Input file is missing: /src/generated/assetsC:mobile-preview.webp (x4)
Input file is missing: /src/generated/assetsC:notfound-404.webp
Input file is missing: /src/generated/assetsC:notfound-404.webp (x2)
Input file is missing: /src/generated/assetsC:notfound-404.webp (x3)
Input file is missing: /src/generated/assetsC:notfound-404.webp (x4)
Input file is missing: /src/generated/assetsC:preview.png
Input file is missing: /src/generated/assetsC:preview.png (x2)
Input file is missing: /src/generated/assetsC:preview.png (x3)
Input file is missing: /src/generated/assetsC:preview.png (x4)
Input file is missing: /src/generated/assetsC:preview.webp
Input file is missing: /src/generated/assetsC:preview.webp (x2)
Input file is missing: /src/generated/assetsC:preview.webp (x3)
Input file is missing: /src/generated/assetsC:preview.webp (x4)
Input file is missing: /src/generated/assetsC:qwer.webp
Input file is missing: /src/generated/assetsC:qwer.webp (x2)
Input file is missing: /src/generated/assetsC:qwer.webp (x3)
Input file is missing: /src/generated/assetsC:qwer.webp (x4)
Input file is missing: /src/generated/assetsC:examplecover.jpg
Input file is missing: /src/generated/assetsC:examplecover.jpg (x2)
Input file is missing: /src/generated/assetsC:examplecover.jpg (x3)
Input file is missing: /src/generated/assetsC:examplecover.jpg (x4)
Input file is missing: /src/generated/assetsC:exampleexample1.jpg
Input file is missing: /src/generated/assetsC:exampleexample1.jpg (x2)
Input file is missing: /src/generated/assetsC:exampleexample1.jpg (x3)
Input file is missing: /src/generated/assetsC:exampleexample1.jpg (x4)
Input file is missing: /src/generated/assetsC:hello-QWERcover.jpg
Input file is missing: /src/generated/assetsC:hello-QWERcover.jpg (x2)
Input file is missing: /src/generated/assetsC:hello-QWERcover.jpg (x3)
Input file is missing: /src/generated/assetsC:hello-QWERcover.jpg (x4)
Input file is missing: /src/generated/assetsC:hello-QWEReach.jpg
Input file is missing: /src/generated/assetsC:hello-QWEReach.jpg (x2)
Input file is missing: /src/generated/assetsC:hello-QWEReach.jpg (x3)
Input file is missing: /src/generated/assetsC:hello-QWEReach.jpg (x4)
Input file is missing: /src/generated/assetsC:hello-QWERwait.jpg
Input file is missing: /src/generated/assetsC:hello-QWERwait.jpg (x2)
Input file is missing: /src/generated/assetsC:hello-QWERwait.jpg (x3)
Input file is missing: /src/generated/assetsC:hello-QWERwait.jpg (x4)
Input file is missing: /src/generated/assetsC:hello-worldcover.jpg
Input file is missing: /src/generated/assetsC:hello-worldcover.jpg (x2)
Input file is missing: /src/generated/assetsC:hello-worldcover.jpg (x3)
Input file is missing: /src/generated/assetsC:hello-worldcover.jpg (x4)
Input file is missing: /src/generated/assetsC:intro-to-QWERcover.jpg
Input file is missing: /src/generated/assetsC:intro-to-QWERcover.jpg (x2)
Input file is missing: /src/generated/assetsC:intro-to-QWERcover.jpg (x3)
Input file is missing: /src/generated/assetsC:intro-to-QWERcover.jpg (x4)
Input file is missing: /src/generated/assetsC:quick-startcover.jpg
Input file is missing: /src/generated/assetsC:quick-startcover.jpg (x2)
Input file is missing: /src/generated/assetsC:quick-startcover.jpg (x3)
Input file is missing: /src/generated/assetsC:quick-startcover.jpg (x4)
Input file is missing: /src/generated/assetsC:avatar.png
Error: Input file is missing: /src/generated/assetsC:avatar.png
Input file is missing: /src/generated/assetsC:avatar.png       
Error: Input file is missing: /src/generated/assetsC:avatar.png
Input file is missing: /src/generated/assetsC:avatar.png
Error: Input file is missing: /src/generated/assetsC:avatar.png
Input file is missing: /src/generated/assetsC:avatar.png       
Error: Input file is missing: /src/generated/assetsC:avatar.png
```

I have added an adapter and confirmed that it builds and deploys correctly on Cloudflare pages.
Therefore, this is an issue only in the local environment.

< various Informations >
- OS： Windows10 Pro 21H2
- IDE: VSCode 1.71.2
- Node: v16.17.0
- NPM: 8.18.0
- svelte-qwer: 0.3.3

※ I am using npm instead of pnpm because I am considering running on Cloudflare

**Comment by kwchang0831** [comment]
Working on it...

**Comment by kwchang0831** [comment]
Thanks so much for your feedback. 🙏

The issue show now be resolved as of [dc98d14](https://github.com/kwchang0831/svelte-QWER/commit/dc98d146d67543808fc2f102bd0163abedb9b0e7).

Sorry for the inconvenience.

**Comment by nemu1113** [comment]
> Thanks so much for your feedback. 🙏
> 
> The issue should now be resolved as of [dc98d14](https://github.com/kwchang0831/svelte-QWER/commit/dc98d146d67543808fc2f102bd0163abedb9b0e7).
> 
> Sorry for the inconvenience.

Hi! @kwchang0831 
Two issues have been resolved and I can confirm that I can build successfully in my local environment.
Thanks for the quick fixes!
